examples/admin_add_user/admin_add_user.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"examples"
+	"fmt"
+	"os"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+)
```

Comment:
Linter shows me an warning here (and in almost every example file)
```File is not `gci`-ed with --skip-generated -s standard -s default```

The correct order would be:
```golang
import (
	"context"
	"fmt"
	"os"

	"examples"

	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
)
```
Please fix the order of imports everywhere it's needed.

Reply:
I ran GCI and it fixed all the linting problems